app/src/main/java/io/github/droidkaigi/confsched/model/Session.java

```diff
@@ -31,9 +34,7 @@
     @SerializedName("description")
     public String description;
 
-    @Column
-    public int speakerId;
-
+    @Column(value = "speakerId", indexed = true)
```

Comment:
Just suggestion of Orma.
I guess it is a little diffecult to understand `value`, and `foreign_key` like ActiveRecord (RoR) is better name.
How do you think?